pages/_app.js

```diff
@@ -38,6 +38,11 @@ export default function App({ Component, pageProps }) {
   //Step 5: Add a title to the graph
   const [titleLabel, setTitleLabel] = useState("");
 
+  //Line Properties
+  const [lineColor, setLineColor] = useState("");
+  const [lineStyle, setLineStyle] = useState("");
+  const [lineWidth, setLineWidth] = useState(0);
```

Comment:
It might be that you want this... but i need to check. Do you want the default value to be 0?